.github/workflows/docs.yml

```diff
@@ -0,0 +1,28 @@
+name: Documents
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  docs:
+    name: Deploy the dev version of documents to GitHub Pages
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
+      - name: Install dependencies
+        run: sudo apt-get install r-base-dev llvm-3.9-dev libclang-3.9-dev
```

Comment:
Do you need the packages `llvm-3.9-dev` and `libclang-3.9-dev`? On ubuntu-20.04, I found I don't need them, and don't need to set `LIBCLANG_PATH` either: https://github.com/clauswilke/helloextendr/blob/main/.github/workflows/R-CMD-check.yaml

(Note that ubuntu-latest is currently 18.04 but will soon switch to 20.04: https://github.com/actions/virtual-environments/issues/1816.)

I think it's better to write action scripts that don't rely on specific versions of llvm/libclang, in particular if they're quite old.

Reply:
Oh, and when ubuntu-latest switches over to 20.04, the script will break because llvm version 3.9 doesn't exist for 20.04.

Reply:
Thanks, I just copied these lines from from the existing YAML and didn't think about the details. Let's remove them.